platforms/common/uORB/uORBDeviceNode.hpp

```diff
@@ -268,15 +275,16 @@ class DeviceNode
 	px4::atomic<unsigned>  _generation{0};  /**< object generation count */
 
 	struct EventWaitItem {
+		hrt_abstime last_update;
 #ifdef CONFIG_BUILD_FLAT
 		class SubscriptionCallback *subscriber;
-#else
-		bool cb_triggered;
 #endif
-		hrt_abstime last_update;
 		uint32_t interval_us;
+		uorb_cb_handle_t next;
 		int8_t lock;
-		uorb_cb_handle_t next; // List ptr
+#ifndef CONFIG_BUILD_FLAT
+		bool cb_triggered;
+#endif
```

Comment:
I could nit about moving this change to the first commit (unrelated to the commit topic). I assume this re-organization is just to prevent the compiler from adding unnecessary padding here ?

Reply:
Yes. I actually didn't intend to leave this change, it was accidentally left in while fiddling with many things at once. I will move it